operations/observability/mixins/IDE/rules/components/openvsx-proxy/alerts.libsonnet

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+ */
+
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gitpod-component-openvsx-proxy-alerts',
+        rules: [
+          {
+            alert: 'GitpodOpenVSXRegistryDown',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '20m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodOpenVsxRegistryDown.md',
+              summary: 'Open-VSX registry is possibly down',
+              description: 'Open-VSX registry is possibly down. We cannot pull VSCode extensions we don\'t have in our caches',
+            },
+            expr:
+              |||
+                sum(rate(gitpod_openvsx_proxy_requests_total{status=~"5\\d\\d|error"}[5m])) > 0.5
+                OR
+                histogram_quantile(0.75, sum(rate(gitpod_openvsx_proxy_duration_upstream_call_seconds_bucket[5m])) by(le)) > 8
```

Comment:
I'm not sure about how to do it properly. I think just alerting on errors should be enough for now. Let's drop it.